libpq/libpq.go

```diff
@@ -0,0 +1,13 @@
+package libpq
+
+import (
+  "fmt"
+  psh "github.com/platformsh/config-reader-go/v2"
+)
+
+// lib/pq requires a specfic connection string to connect to PostgreSQL.
+func FormattedCredentials(creds psh.Credential) (string, error) {
+  formatted := fmt.Sprintf("host=%s port=%d user=%s " + "password=%s dbname=%s sslmode=disable",
```

Comment:
Why is this string split in 2?  It should be perfectly safe to make a single string, no?